CHANGELOG.asciidoc

```diff
@@ -48,6 +48,7 @@ https://github.com/elastic/apm-server/compare/6.1...6.2[View commits]
 - Increase default 'ConcurrentRequests' from 20 to 40 {pull}492[492]
 - Change config option `frontend.sourcemapping.index` to `frontend.source_mapping.index_pattern` and remove adding a '*' by default.{pull}492[492].
 - Add Config option for excluding stack trace frames from `grouping_key` calculation {pull}482[482]
+- Remove config files from beats. Manually add relevant config options {pull}578[578]
```

Comment:
There is a changelog entry already from PR #496 .

> - Remove untested config options from config yml files {pull}496[496]

The manual changes were made before the script to update beats was changed, but basically had the same effect. As you can see from the changes in the apm config yml files in this PR it is basically cosmetic changes, but no config options from beats are removed, as they were already before. 
So maybe a more accurate entry would be only `Remove config files from beats.`